src/ipf.cpp

```diff
@@ -351,8 +351,8 @@ static void ReadTrack (t_track *pt_)
   if (!pt_->data)
   {
     // Allocate enough for the full decoded size, allowing for expanded overlapping sectors
-    // TODO: This is not freed. Use a unique_ptr or shared_ptr to make it cleaner. 
-    memcpy(pt_->data = new byte[uDecoded], abDecoded, pt_->size = uDecoded);
+    std::shared_ptr<byte[]> datamng = std::shared_ptr<byte[]>(new byte[uDecoded]);
```

Comment:
Missing `#include <memory>`.
You need to use `make WITH_IPF=true` to test (but you'll need to download IPF libs before, cf `.travis.yml` for an example).

Reply:
I'm also a bit worried that I don't have any test for IPFs, but the problem here I don't have a way to generate an IPF with a custom file in it and I don't want to include a real game in the source tree for obvious reasons ...

Reply:
ah right because I use clang :)